product_packaging_unit_price_calculator/security/ir.model.access.csv

```diff
@@ -1,2 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_product_package_price_wizard,access.product.package.price.wizard,model_product_package_price_wizard,stock.group_stock_user,1,1,1,1
+access_product_package_price_wizard,access.product.package.price.wizard,model_product_package_price_wizard,base.group_system,1,1,1,0
```

Comment:
it means that nobody except `admin` has the right to unlink those?

Reply:
Transient model are removed by a cron job, and every access rights configured on wizard in other standard addons have these access rights.